staff/groups/aleti/github-app-2.0.0-react/src/components/SearchUsers.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './SearchUsers.css'
+
+function SearchUsers(props) {
```

Comment:
TIP following comments before, make this compo smart one, managing query state, and flow it up to parent on submit

Reply:
+1